src/RoutingControllersOptions.ts

```diff
@@ -85,4 +86,28 @@ export interface RoutingControllersOptions {
      */
     currentUserChecker?: CurrentUserChecker;
     
+    /**
+     * Default settings
+     */
+    defaults?: {
+        /**
+         * If set, all null responses will return specified status code by default
+         */
+        nullResultCode?: number;
+
+        /**
+         * If set, all undefined responses will return specified status code by default
+         */
+        undefinedResultCode?: number;
+
+        /**
+         * Default param options
+         */
+        paramOptions?: {
```

Comment:
But in the future we might need to introduce new param options (transform/validate?), so I think it's better if it's extendable so we wouldn't have to introduce breaking change.
 `paramsAlwaysRequired: true,` isn't much shorter than `paramOptions: { required: true },` 😉